platforms/core-runtime/base-services/src/main/java/org/gradle/internal/jvm/JpmsConfiguration.java

```diff
@@ -85,7 +85,8 @@ public class JpmsConfiguration {
             "--add-opens=java.base/java.net=ALL-UNNAMED", // required by JavaObjectSerializationCodec
             "--add-opens=java.base/java.util.concurrent=ALL-UNNAMED", // required by AccessTrackingProperties
             "--add-opens=java.base/java.util.concurrent.atomic=ALL-UNNAMED", // serialized from org.gradle.internal.file.StatStatistics$Collector
-            "--add-opens=java.xml/javax.xml.namespace=ALL-UNNAMED" // serialized from IvyDescriptorFileGenerator.Model
+            "--add-opens=java.xml/javax.xml.namespace=ALL-UNNAMED", // serialized from IvyDescriptorFileGenerator.Model
+            "--add-opens=java.base/java.time=ALL-UNNAMED"
```

Comment:
💅 Could you please add a comment like `// required by JavaObjectSerializationCodec`.

Reply:
I'll added it altogether with the test coverage